lesson_02/job1/main.py

```diff
@@ -18,7 +18,7 @@
 app = Flask(__name__)
 
 
-@app.route('/', methods=['POST'])
+@app.route('/', methods=['GET'])
```

Comment:
This endpoint should accept the POST request because it's creating something. 
GET should be used when you get an information about something or some data